server/liveblog/themes/themes_assets/amp/gulpfile.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+
+var DEBUG = process.env.NODE_ENV !== "production";
+var defaultThemePath = './node_modules/liveblog-default-theme/';
+var testdata = require(defaultThemePath + 'test');
+
+// Requires.
+var gulp = require('gulp')
+  , gulpLoadPlugins = require('gulp-load-plugins')
+  , plugins = gulpLoadPlugins()
+  , path = require('path')
+  , eslint = require('gulp-eslint')
+  , fs = require('fs')
+  , nunjucks = require('nunjucks')
+  , dateFilter = require('nunjucks-date-filter')
+  , purify = require('gulp-purifycss')
+  , cleanCSS = require('gulp-clean-css')
```

Comment:
plugins.purify and plugins.cleanCSS are undefined

[15:55:11] TypeError: plugins.purify is not a function
[15:56:52] TypeError: plugins.cleanCSS is not a function